pkg/render/manager.go

```diff
@@ -109,17 +111,28 @@ func Manager(cfg *ManagerConfiguration) (Component, error) {
 
 // ManagerConfiguration contains all the config information needed to render the component.
 type ManagerConfiguration struct {
-	KeyValidatorConfig      authentication.KeyValidatorConfig
-	ESSecrets               []*corev1.Secret
-	TrustedCertBundle       certificatemanagement.TrustedBundle
-	ESClusterConfig         *relasticsearch.ClusterConfig
-	TLSKeyPair              certificatemanagement.KeyPairInterface
-	PullSecrets             []*corev1.Secret
-	Openshift               bool
-	Installation            *operatorv1.InstallationSpec
-	ManagementCluster       *operatorv1.ManagementCluster
-	TunnelSecret            certificatemanagement.KeyPairInterface
-	InternalTrafficSecret   certificatemanagement.KeyPairInterface
+	KeyValidatorConfig authentication.KeyValidatorConfig
+	ESSecrets          []*corev1.Secret
+	ESClusterConfig    *relasticsearch.ClusterConfig
+	PullSecrets        []*corev1.Secret
+	Openshift          bool
+	Installation       *operatorv1.InstallationSpec
+	ManagementCluster  *operatorv1.ManagementCluster
+
+	// If provided, the KeyPair to used for external connections to Voltron.
```

Comment:
I am not sure this comment is correct. I think need it for external connections for users that access the UI ? My memory could play tricks on me though ...

Reply:
This is one of the confusing things about how this works (and that we will be fixing in the near future).

Voltron is both the tunnel manager, and ALSO the reverse-proxy for the Manager UI - all requests to the UI come in through Voltron (which is run within the manager pod).

So Voltron is technically the correct word right now - Voltron terminates the TLS connection and proxies it using HTTP (not encrypted) to localhost within the pod, where the manager nginx container ultimately handles the request.

Reply:
The comment is slightly wrong as of this PR since it is also used for es-proxy -> Linseed